docs/source/compiler.rst

```diff
@@ -36,7 +36,7 @@ You can initialize a local ``quilc`` server by typing ``quilc -S`` into your ter
     ... - Launching quilc.
     ... - Spawning server at (tcp://*:5555) .
 
-To get a description of ``quilc``, and options and examples of its command line use, see :ref:`quilc_man`.
```

Comment:
The `quilc_man` ref no longer exists, as far as I can tell.

Reply:
Maybe also refer to the README as I did here: https://github.com/rigetti/pyquil/pull/818/files#diff-b5a59ade87b85f1812758e9f3b8f400fR91

Reply:
Done.